packages/algob/src/lib/tx.ts

```diff
@@ -266,20 +268,70 @@ export async function makeAndSignTx (
   return [txns, signedTxn];
 }
 
+/**
+ * Signs transaction object(s) and returns raw signed transaction
+ * Note: Sign transaction is used to sign single transaction and `signTransactions` takes
+ * SDK transaction object, signer and signs it.
+ * @param txnAndSign Transaction object(s) with signers
+ */
+export function signTransactions (
+  txnAndSign: wtypes.TransactionAndSign[]
+): Uint8Array | Uint8Array[] {
```

Comment:
After the update, the return type is always array
```suggestion
): Uint8Array[] {
```